src/engine/cuecontrol.cpp

```diff
@@ -220,7 +217,8 @@ void CueControl::trackLoaded(TrackPointer pNewTrack) {
         }
         int hotcue = pCue->getHotCue();
         if (hotcue != -1) {
-            attachCue(pCue, hotcue);
+            HotcueControl* pControl = m_hotcueControls.value(hotcue, nullptr);
```

Comment:
Accessing the items of m_hotcueControls should be wrapped into a function that deals with returning nullptr if the index is out of range. Duplicating this code is highly unsafe. Creating a QList for items without a well-defined default constructor should be forbidden!